File: src/main/java/pl/szczypkowski/vehiclesfleetmanager/map/model/Coordinates.java
package pl.szczypkowski.vehiclesfleetmanager.map.model;

import javax.persistence.*;

@Entity
@Table(name = "coordinates")
public class Coordinates {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "lon")
    private String lon;
    @Column(name = "lat")
    private String lat;
    @Column(name = "details")
    private String details;
    @Column(name="color")
    private String color;
    @Column(name = "name")
    private String name;


    public Coordinates() {
    }

    public Coordinates(Long id, String lon, String lat, String details) {
        this.id = id;
        this.lon = lon;
        this.lat = lat;
        this.details = details;
    }


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getDetails() {
        return details;
    }

    public void setDetails(String details) {
        this.details = details;
    }

    public String getLon() {
        return lon;
    }

    public void setLon(String lon) {
        this.lon = lon;
    }

    public String getLat() {
        return lat;
    }

    public void setLat(String lat) {
        this.lat = lat;
    }

    @Override
    public String toString() {
        return "Coordinates{" +
                "id=" + id +
                ", lon='" + lon + '\'' +
                ", lat='" + lat + '\'' +
                ", details='" + details + '\'' +
                '}';
    }
}

File: src/main/web/VehiclesFleetManagerFront/src/app/model/coordinates.ts
export class Coordinates {
  constructor(_lat: number | undefined, lon: number | undefined) {
    this._lat=_lat;
    this._lon=lon;

  }


  _lon: number | undefined;
   _lat: number | undefined;
  details: string | undefined;
  color: string | undefined;
  name: string | undefined;

  get lon(): number | undefined {
    return this._lon;
  }

  set lon(value: number | undefined) {
    this._lon = value;
  }

  get lat(): number | undefined {
    return this._lat;
  }

  set lat(value: number | undefined) {
    this._lat = value;
  }
}
